source-repos.sh

```diff
@@ -90,7 +90,19 @@ ES_RABBITMQ_LOGS=`git log --after=$DATE -- \
   . `
 popd
 
-echo "$E_LOGS \n $EC_LOGS \n $ES_DISCOVERY_LOGS \n $ES_SAMPLE_LOGS \n $ES_CEPH_LOGS \n $ES_SQS_LOGS \n $ES_PROM_LOGS \n $ES_GITLAB_LOGS \n $ES_GITHUB_LOGS \n $ES_COUCHDB_LOGS \n $ES_CAMEL_LOGS \n $ES_RABBITMQ_LOGS" \
+git clone https://github.com/knative-sandbox/eventing-redis.git
+pushd $KN_SANDBOX_PATH/eventing-redis
+ES_REDIS_LOGS=`git log --after=$DATE -- \
+  source `
```

Comment:
I am not sure `source` is correct, looks like this should be a `.`

Reply:
ooh, are you selecting the source directory? okok nevermind.

Reply:
Yeah, I was thinking of doing something like that for rabbitmq as well (it currently counts broker contrib as well), but since the layout has some of the code in pkg as well, seems fine for now.